common/src/Microsoft.Azure.IIoT.Abstractions/src/Processor/IMonitoredItemSampleProcessor.cs

```diff
@@ -3,8 +3,8 @@
 //  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
-namespace Microsoft.Azure.IIoT.OpcUa.Subscriber {
-    using Microsoft.Azure.IIoT.OpcUa.Subscriber.Models;
+namespace Microsoft.Azure.IIoT.Processor {
```

Comment:
I would very much prefer this stuff to stay in Opc.Ua, but move into the Opc.Ua root package (aka opc ua abstractions).   Instead of moving tot he common abstraction.   It will be difficult to sort out later and it is at this point very much OPC UA.

Reply:
done